src/capi/image.rs

```diff
@@ -184,20 +184,18 @@ impl From<&Image> for avifImage {
             if !image.has_plane(i.into()) {
                 continue;
             }
-            let plane = image.plane(i.into()).unwrap();
-            dst_image.yuvPlanes[i] = if image.depth == 8 {
-                plane.data.unwrap().as_ptr() as *mut u8
+            dst_image.yuvPlanes[i] = if image.depth > 8 {
```

Comment:
this is an interesting way to spell a ternary.

Reply:
if you look at this as a ternary, then yes i agree it look rather weird. but i think this is more than just a ternary. we could have fully functional code in each of the `if` and `else` blocks and the last statement in each block will be the value that is assigned. so i think it is useful that way. we can also use `match` statements this way.

i think the main reason for this type of a syntax is because in rust everything must be initialized. so this is better than declaring a variable and initializing it separately in the if and else blocks.